nemo/collections/nlp/metrics/classification_report.py

```diff
@@ -97,6 +97,7 @@ def get_precision_recall_f1(
         recall = torch.where(tp + fn != zeros, tp / (tp + fn) * 100, zeros)
         f1 = torch.where(precision + recall != zeros, 2 * precision * recall / (precision + recall), zeros)
 
+        print("")
```

Comment:
did it fix the first row alignment issue?

Reply:
Yes, but would investigate the other model to see why it happens just for my model.